pandas/tests/indexes/interval/test_constructors.py

```diff
@@ -38,25 +40,41 @@ class ConstructorTests:
     get_kwargs_from_breaks to the expected format.
     """
 
+    def _skip_test_constructor(self, dtype):
+        # get_kwargs_from_breaks in TestFromTuples and TestClassconstructors just return
+        # tuples of ints, so IntervalIndex can't know the original dtype
```

Comment:
Is the tuple of ints key for the constructor tests or can they be cast to an array where the dtype is known?